docs-translations/zh-CN/glossary.md

```diff
@@ -0,0 +1,96 @@
+# Glossary
+
+这篇文档说明了一些经常在 Electron 开发中使用的专业术语。
+
+### ASAR
+
+ASAR 代表了 Atom Shell Archive Format。一个 [asar][asar] 压缩包就是一个简单的 `tar` 文件-就像将那些有联系的文件格式化至一个单独的文件中。Electron 能够任意读取其中的文件并且不需要解压缩整个文件。
```

Comment:
The `asar` link doesn't work here. I think the original document is missing an `asar` url link at bottom of the document, you can add it in this translated document.